ui/src/RoomManage.tsx

```diff
@@ -38,7 +39,7 @@ const defaultMode = (authMode: UIConfig['authMode'], loggedIn: boolean): RoomMod
 };
 
 const CreateRoom = ({room, config}: Pick<UseRoom, 'room'> & {config: UIConfig}) => {
-    const [id, setId] = React.useState(randomRoomName);
+    const [id, setId] = useRoomID(randomRoomName());
```

Comment:
I don't think useRoomId is the right hook here, this hook is will add a browser navigation state on every state change meaning you can use the back button in your browser to revert each keystroke.

Instead of using randomRoomName as default we can use the method from useRoomID to extract the currently selected roomId from the url like this:

```typescript
const [id, setId] = React.useState(() => getRoomFromURL(window.location.search) ?? randomRoomName());
```

Reply:
ah I see, thanks for the hint!